### PR TITLE
Addition of "Advanced Data Operations"

### PR DESCRIPTION
**Pull request by brad302**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by brad302** [comment]
Just a quick one, the ISV submission has "application/text" rather than "text/plain" so that will need to be corrected for consistency.  Will need assistance on how to do that.  I see that I'm not able to change the submission once it's been created.

**Comment by v-EgorKozhadei** [comment]
Hello @brad302, 

Thank you very much and congratulations on your new submission. Our Certification Engineers will review it within 1-2 weeks. 

You'll be able to update the submission in ISV Studio by using the latest commit ID once your Pull Request is approved and merged.

Please let us know if you have any questions. 

Thank you very much again for working with us.

File: certified-connectors/Advanced Data Operations/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1389 @@
...
+        },
+        "schema": {
+          "$ref": "#/definitions/schema"
+        },
+        "advancedOptions": {
+          "$ref": "#/definitions/advancedOptions"
+        },
+        "data": {
+          "type": "string",
+          "description": "Base64 encoded string  containing contents of a parquet data file.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Advanced Data Operations/apiDefinition.swagger.json:1351]: @brad302 please make sure that descriptions are grammatically correct and are free of typographical errors. There's an extra whitespace in this string. Please refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information.

File: certified-connectors/Advanced Data Operations/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Advanced Data Operations
+
+A suite of advanced API's designed to assist in the transformation of bulk datasets.
+
```
Comment (v-EgorKozhadei) [inline certified-connectors/Advanced Data Operations/README.md:4]: @brad302 please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).

File: certified-connectors/Advanced Data Operations/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1389 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Advanced Data Operations",
+    "version": "1.0.0",
+    "description": "A suite of advanced API's designed to assist in the transformation of bulk datasets.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Advanced Data Operations/apiDefinition.swagger.json:6]: @brad302 please remove the word `API` or replace it with other word (e.g. services) in the description,

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @brad302,

Please check comments after the review. There are a few minor things that need to be fixed.

Let us know if you have any questions.

File: certified-connectors/Advanced Data Operations/apiDefinition.swagger.json (hunk 1 above)
Comment (brad302) [reply certified-connectors/Advanced Data Operations/apiDefinition.swagger.json]: Thanks @v-EgorKozhadei, that's done.

File: certified-connectors/Advanced Data Operations/README.md (hunk 2 above)
Comment (brad302) [reply certified-connectors/Advanced Data Operations/README.md:7]: Thanks @v-EgorKozhadei, that's done.  I hope it now confirms.

File: certified-connectors/Advanced Data Operations/apiDefinition.swagger.json (hunk 3 above)
Comment (brad302) [reply certified-connectors/Advanced Data Operations/apiDefinition.swagger.json]: Thanks @v-EgorKozhadei, that's done.

**Review by v-EgorKozhadei — APPROVED** [review]

**Pull request merged**